.github/workflows/test.yml

```
@@ -9,12 +9,12 @@ on:
   workflow_call:
 
 jobs:
-  lint_markdown:
+  test:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
       - uses: actions/setup-node@v3
         with:
           cache: npm
       - run: npm ci
-      - run: npm run lint:markdown
+      - run: npm run test
```

Having independent steps could make CI results easier to read 🙂 
```suggestion
      - run: npm run lint:markdown
      - run: npm run lint:css
```